src/device-manager/java/Makefile.am

```diff
@@ -110,6 +110,8 @@ WeaveDeviceManager_jar_JAVA_SRCS                   =        \
     nl/Weave/DataManagement/GenericTraitUpdatableDataSinkImpl.java \
     nl/Weave/DataManagement/WdmClientFactory.java           \
     nl/Weave/DataManagement/ResourceIdentifier.java         \
+	nl/Weave/DataManagement/WdmClientFlushUpdateException.java          \
```

Comment:
nit: looks like the indentation is off here -- should be 2 spaces to the left

Reply:
Done, thanks.